pkg/generators/extension_test.go

```diff
@@ -136,69 +141,58 @@ func TestExtensionErrors(t *testing.T) {
 
 	var tests = []struct {
 		comments []string
-		err      error
 	}{
 		{
 			// Missing extension value should be an error.
 			comments: []string{
 				"+k8s:openapi-gen=x-kubernetes-no-value",
 			},
-			err: errors.New("Invalid extension value: x-kubernetes-no-value"),
 		},
 		{
 			// Wrong separator should be an error.
 			comments: []string{
 				"+k8s:openapi-gen=x-kubernetes-wrong-separator;error",
 			},
-			err: errors.New("Invalid extension value: x-kubernetes-wrong-separator;error"),
 		},
 		{
 			// disallowed is not one of the allowed values for listType.
 			comments: []string{
 				"+listType=disallowed",
 			},
-			err: errors.New("listType: value(s) [disallowed] not allowed. Allowed values: [atomic map set]\n"),
```

Comment:
OTOH putting the whole string here is useful because it lets us ask questions like, "why does this error message have a trailing newline?" :)

Reply:
Removed newlines from error messages.